cmds/build

```diff
@@ -28,14 +28,14 @@ build_main() {
         fi
 
         local entry=(${l})
-        local machine="${entry[0]}"
-        local image="${entry[1]}"
+        local image="${entry[0]}"
+        local env="${entry[@]:1}"
 
         if [ "${mode}" = "clean" ]; then
-            MACHINE="${machine}" bitbake -c cleanall "${image}"
+            eval ${env} bitbake -c cleanall "${image}"
```

Comment:
That's is presumed acceptable since `${env}` is read from the versioned manifest files.